### PR TITLE
upgrades for ports libewf and sleuthkit

### PR DESCRIPTION
**Pull request by RJVB**
Working on adding lzvn support to my afsctool version I noticed these ports were outdated, and upgraded them.

Not looking to become their maintainer, just to share the fruits of getting side-tracked.

### CODE REVIEW

File: security/libewf/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        libyal libewf 20140608
+github.setup        libyal libewf 20171104
 revision            1
```
Comment (pmetzger) [inline security/libewf/Portfile]: The revision can go away since we're bumping version.

File: security/libewf/Portfile [hunk 2]
```diff
@@ -18,10 +18,13 @@ long_description    Libewf is a library for support of the Expert Witness \
                     Libewf allows you to read and write media information \
                     within the EWF files.
 
-master_sites        https://raw.githubusercontent.com/libyal/legacy/master/libewf/
+master_sites        https://github.com/libyal/libewf/releases/download/${version}/
+distname            ${name}-experimental-${version}
+worksrcdir          ${name}-${version}
```
Comment (pmetzger) [inline security/libewf/Portfile]: The github portgroup can't be made use of?

File: security/libewf/Portfile [hunk 3]
```diff
@@ -32,9 +35,21 @@ depends_lib         port:bzip2 \
                     path:lib/pkgconfig/fuse.pc:osxfuse \
                     port:zlib
 
+pre-configure {
+    use_autoreconf  [expr ![file exists ${worksrcpath}/configure]]
```
Comment (pmetzger) [inline security/libewf/Portfile]: Either it exists always in this release or it doesn't, right?

File: sysutils/sleuthkit/Portfile [hunk 4]
```diff
@@ -24,20 +23,19 @@ long_description    The Sleuth Kit (previously known as TASK) is a collection \
 
 homepage            http://www.sleuthkit.org/sleuthkit/
 license             cpl ibmpl
-master_sites        sourceforge:project/sleuthkit/sleuthkit/${version}
 
-checksums           rmd160  223c6ffe22259ca057b6d9634813536e7ccd9dba \
-                    sha256  67f9d2a31a8884d58698d6122fc1a1bfa9bf238582bde2b49228ec9b899f0327
+github.setup        sleuthkit sleuthkit 4.6.2 sleuthkit-
+checksums           rmd160  e993cb5cc1a8bd730a846c72ca73078e54e94248 \
+                    sha256  40cd4c2feded02707dccb85a92bdc9e5b3cd70269aa4c46bee27b7567bad163d
```
Comment (pmetzger) [inline sysutils/sleuthkit/Portfile]: We'll also want "size"...

**Comment by kencu** [comment]
I think I can address these issues up for you, @RJVB -- give me a day or so.

**Comment by kencu** [comment]
there are two different ports here, and both needed a bit 'o tweaking. So they are not separate commits anymore. I don't use osxfuse for anything, and have no idea how to test these ports. But I think what I tweaked here address the issues you raised. Too bad I couldn't tweak them in a way to force push the tweaks into each separate commit. My git-fu is lacking, zen master.

If I used these ports and want to update them, I'd probably have to break these out and do them separately.

**Comment by kencu** [comment]
I'm going to close this, because I can't separate these two port updates into separate PRs. I made two new PRs, one for each, and we'll work on those instead. Thanks, Rene.

**Pull request closed without merging**